docs/guide/release-notes/1.3.0.md

```diff
@@ -0,0 +1,27 @@
+---
+title: lit-html 1.3.0
+slug: lit-html-1-3-0
+tags: release
+---
+
+See the [changelog](https://github.com/Polymer/lit-html/blob/master/CHANGELOG.md) for a complete list of changes.
+
+## Trusted Types
+
+[Trusted Types](https://web.dev/trusted-types/) helps eliminate cross-site scripting (XSS) attacks by limiting the ability to use unsafe DOM APIs, like `innerHTML`. When trusted types is enabled, unsafe sinks disallow use with easily user-controlled types like strings, and instead require values be one of a small set of trusted types, like `TrustedHTML` that can only be created by a Trusted Type policy.
+
+This prevents application code from accidentally passing unsanitized user-controlled values directly to unsafe sinks and makes it easier to review security sensitive code that uses the Trusted Type policies.
+
+lit-html 1.3.0 is compatible with enabling Trusted Types. lit-html was already very XSS-resistant because it treats strings as _text_ content, not unsafe HTML. It does have to the HTML from template strings to unsafe sinks like `innerHTML`, and now it does this via a Trusted Type policy if the API exists.
```

Comment:
It does have to the HTML -> It does have to parse the HTML

Reply:
Not sure what this paragraph is trying to say. Maybe, "It does have the ability to pass HTML from template strings to unsafe sinks ... "?

The second sentence appears to contradict the first. Is the second sentence talking about using things like `unsafeHTML`?

Also, maybe include a link to some information on Trusted Types? Maybe https://web.dev/trusted-types/ ?

Reply:
Oh doh. Please ignore the last comment.